benchopt/utils/slurm_executor.py

```diff
@@ -0,0 +1,50 @@
+import yaml
+
+try:
+    import submitit
+    from submitit.helpers import as_completed
+    from rich import progress
+except ImportError:
+    raise ImportError(
+        "Benchopt needs submitit and rich to launch computation on a SLURM "
+        "cluster. Please use `pip install submitit rich` to use the --slurm "
+        "option."
+    )
+
+
+def get_executor_jzay(slurm_config, timeout=100, job_name="benchopt_run"):
+
+    with open(slurm_config, "r") as f:
+        config = yaml.safe_load(f)
+
+    # It the job timeout is not specified in the config file, use 1.5x the
+    # benchopt timeout. This value is a trade-off between helping the
+    # scheduler (low slurm_time allow for faster accept) and avoiding
+    # killing the job too early.
+    if 'slurm_time' not in config:
+        # Timeout is in second in benchopt
+        config['slurm_time'] = f"00:{int(1.5*timeout)}"
```

Comment:
did you test this for high timeouts?

Reply:
I tested this with timeout >60 (at least 450) and this is working. Is that what you had in mind or larger stuff?

Reply:
no exactly what I had in mind